test/e2e/config/e2e_conf.yaml

```diff
@@ -141,15 +141,15 @@ variables:
   CONFIG_FILE_PATH: "${HOME}/.cluster-api/clusterctl.yaml"
   SERVICE_CIDR: "10.96.0.0/12"
   POD_CIDR: "192.168.0.0/18"
-  PROVISIONING_CIDR: "24"
+  BM_PROVISIONER_CIDR: "24"
   CLUSTER_APIENDPOINT_HOST: "192.168.111.249"
   CLUSTER_APIENDPOINT_PORT: "6443"
   EXTERNAL_SUBNET_V4_PREFIX: "24"
   CLUSTER_PROVISIONING_INTERFACE: "ironicendpoint"
-  BAREMETALV4_POOL_RANGE_START: "192.168.111.100"
```

Comment:
why external and not baremetal?

Reply:
In our existing code base, the network we refer to as BAREMETAL is in reality the external network , same interface is used for K8s API for example.